firefox.yaml

```diff
@@ -94,11 +78,8 @@ pipeline:
       # with release repositories, so just extract what it does and do it
       # manually.
       # https://firefox-source-docs.mozilla.org/setup/linux_build.html
-      # Clone the repository. If the major version is changed, this needs to
-      # be updated.
-      echo "RUNNING: hg clone https://hg.mozilla.org/releases/mozilla-release/ -u ${RELEASE_TAG}"
-      hg clone https://hg.mozilla.org/releases/mozilla-release/ -u ${RELEASE_TAG}
-      cd mozilla-release
+      wget https://ftp.mozilla.org/pub/firefox/releases/${{package.version}}/source/firefox-${{package.version}}.source.tar.xz -O- | tar Jx
+      cd firefox-${{package.version}}
```

Comment:
Instead of using `wget` directly, we can use the fetch pipeline here

Reply:
Done, thank you!